torch/onnx/_internal/fx/op_validation.py

```diff
@@ -22,6 +22,25 @@
 
 
 @_beartype.beartype
+def _op_level_debug_message_formatter(
+    fn: Callable,
+    self,
+    node: torch.fx.Node,
+    symbolic_fn: Union[onnxscript.OnnxFunction, onnxscript.TracedOnnxFunction],
+    *args,
+    **kwargs,
+) -> str:
+    return (
+        f"FX Node: {node.op}:{node.target}[name={node.name}]. \n"
+        f"ONNX Node: {symbolic_fn.name}."
```

Comment:
nit: I don't recall if this already does, but we might want to include domain as well? I think right now they are all torchlib, but just in case of custom domain.

Reply:
I added opset info